src/_help/publish-documentation/deploy-and-release-management.md

```diff
@@ -0,0 +1,85 @@
+
+After processing, the new API document will be automatically deployed after a few moments unless you have enabled Manual Release mode.
+
+### Deploy from the CLI
+
+Our CLI can also help with deploying an API document using the `deploy` command. The complete process is available on the dedicated [CLI page](/help/continuous-integration/index/).
+
+### Deploy using the GitHub Action
+
+Our [GitHub Action](/help/continuous-integration/github-actions/) allows you to easily integrate Bump.sh into your projects by simply adding a workflow file.
+
+## Release management
+
+Publishing a new API definition of your documentation typically follows a two-step process:
+- You upload a new API definition of your documentation through an API document, your specification file.
+- This new API document is validated and deployed, becoming the "visible" API definition of your documentation. The changelog will also be update, reflecting changes between the current and previous definitions.
+
+By default, these two steps are automatically linked: uploading a new API document results in its publication. However, it is possible to dissociate them using the Manual Release mode.
+
+The Deployments section of your documentation provides access to the history of all API documents uploaded on Bump.sh, including their status (`enqueued`, `deploying`, `deployed`, or `errored`) and their release status (`Released` or `Not released`).
+
+![](/images/help/deployments-list.png)
+
+### Manual Release
+
+The Manual Release mode is still in beta. Please don't hesitate to contact us if you have any questions.
+
+The Manual Release mode is an option that allows you to separate the upload step from the publication step of your API documentation.
+
+![](/images/help/manual-release-toggle.png)
+
+When activated, uploading an API document of your API documentation will no longer automatically publish it. We continue to prepare this API definition on our side (and you can view a preview and its changelog), but the release step becomes manual. This means you can decide when to make an API definition live for your API users.
+
+This option allows you to continue working on your documentation without worrying about the impact on your users. For example, you can prepare a future definition of your API and its documentation with members of your organization and launch its publication when the time is right.
+
+#### Manually release a deployment
+
+From the list of your deployments, select an API document of your documentation with the "unreleased" status.
+
+![](/images/help/deployments-list-not-released.png)
+
+The deployment preparation page will appear, allowing you to add context by adding a title or a brief description, for example, to accompany this new definition. These fields [support Markdown](/help/specification-support/markdown-support/) to make the result more readable.
+
+![](/images/help/deployment-release-form.png)
+
+### Unrelease an API definition
```

Comment:
I'm not 100% sure of this one. It's more consistent with the previous title, but it doesn't 100% work. Please ignore it if you think it's better the way it's written now.
```suggestion
### Unrelease a deployment
```